seqr/views/apis/summary_data_api.py

```diff
@@ -286,39 +286,45 @@ def _add_row(row, family_id, row_type):
         elif row_type == DISCOVERY_ROW_TYPE:
             for i, discovery_row in enumerate(row):
                 participant_id = discovery_row.pop('participant_id')
-                parsed_row = {'{}-{}'.format(k, i + 1): v for k, v in discovery_row.items()}
+                parsed_row = {'{}-{}'.format(k, i + 1): v for k, v in discovery_row.items() if k != 'allele_balance_or_heteroplasmy_percentage'}
```

Comment:
what's the significance of excluding `allele_balance_or_heteroplasmy_percentage` here?

Reply:
its not used in the report and as part of this cleanup I am trying to stop returning fields that are unused. The effort to configure the function that generates the rows to omit this field was too clunky/ difficult so this seemed clearer